src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -42,10 +44,26 @@ public String saveTicket(Ticket ticket){
             log.info(e.toString());
             return "error";
         }
-
         return "start";
     }
 
+    @RequestMapping("/uploadImage")
+    public String uploadImage( @RequestParam("imageFile") MultipartFile imageFile, Model model) {
+        String  returnValue = "start";
+        try {
+            ticketService.saveImage(imageFile);
+            Ticket ticket = new Ticket();
+            model.addAttribute( "ticket", ticket);
+            returnValue = "start";
+        } catch (IOException e) {
+            e.printStackTrace();
+            returnValue = "error";
+        }
+
+
+        return returnValue;
+    }
+
```

Comment:
This was based off what I learn, in hope that it can serve you here since you guys were wanting this.

Reply:
Is it used anywhere?  If not, it's an incomplete feature, which is technical debt.